content/devs.json

```diff
@@ -184,6 +190,39 @@
       }
     ]
   },
+  {
+    "avatar": "https://i.imgur.com/LXw7ryd.jpg",
+    "name": "Harvey Jay M. Sison",
+    "title": "Platform Engineer",
+    "company": "Exora Technologies Incorporated",
+    "blurb": "Hi! 🙌🏼 I'm a full stack software engineer 💻, a UX/UI designer🎨, and an aspiring technopreneur 👨🏽‍💼  from Quezon City, Philippines",
+    "skills": [
+      "ReactJS",
+      "NodeJS + Express",
```

Comment:
```sh
Harvey Jay M. Sison has invalid details on devs.json
[ { message:
     '"skills[1]" length must be less than or equal to 15 characters long',
    path: [ 'skills', 1 ],
    type: 'string.max',
    context:
     { limit: 15,
       value: 'NodeJS + Express',
       encoding: undefined,
       label: 'skills[1]',
       key: 1 } } ]
```

You can separate NodeJS and Express as skills.

Reply:
Separated it. Thank you